src/AspNetCoreModuleV2/AspNetCore/src/precomp.cpp

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I think you need to include precomp.h in this file.

Reply:
It is included by `ForcedIncludeFiles`.